evoting/lib/election.go

```diff
@@ -34,9 +34,10 @@ type Election struct {
 	Key    kyber.Point           // Key is the DKG public key.
 	Stage  uint32                // Stage indicates the phase of the election.
 
-	Data        []byte // Data can hold auxiliary information.
-	Description string // Description in string format.
-	End         string // End (termination) date.
+	Candidates  []uint32 // Candidates is the list of candidate scipers.
+	MaxChoices  uint32   // MaxChoices is the max votes in allowed in a ballot.
+	Description string   // Description in string format.
+	End         string   // End (termination) date.
```

Comment:
In Go, normally this would always be a time.Time. The only exception is at the edge of a system, where you are interacting with another system (i.e. using protobuf to send this via to JS?). So, does this one qualify for the exception, or are you going to make this time.Time, like Rob Pike intended? :)

Reply:
I would do both as `int64` and comment that they are in unix time format...

Reply:
👍 changed in the new commit.